src/main/kotlin/com/hibob/academy/employeeFeedback/dao/FeedbackDao.kt

```diff
@@ -104,9 +104,10 @@ class FeedbackDao(private val sql: DSLContext) {
             .fetch(feedBackMapper)
     }
 
-    fun updateStatus(status: Status, feedbackId: Long, companyId: Long): Int =
+    fun updateStatus(companyId: Long,status: Status, feedbackId: Long): Int =
```

Comment:
nice that you moved the companyId to be the 1st param 👍 .
Notice that after "," there should be a space, so remember to format your code